samples/js/splore/src/main.js

```diff
@@ -52,31 +64,45 @@ function load() {
     });
   }
 
-  if (!params.db) {
-    renderPrompt();
+  if (params.p && params.db) {
```

Comment:
Not if we want Splore to be able to render a database root.

Generally speaking, given a spec, we can't tell if it's for a path or a db. Splore needs to be told what it's looking for.

If you want to drop the feature of inspecting a database... well I find that kind of handy, if awkward in Splore. @aboodman